app/controllers/availabilities_controller.rb

```diff
@@ -20,15 +20,17 @@ def new
                                      end_time: event.end_time)
     @availability.person = person if person
 
-    @people_collection = Person.active
-    @people_collection |= [person]
+    people = Person.active
+    people |= [person]
+    @people_collection = people.sort_by {|firstname, middleinitial, lastname| "#{firstname} #{middleinitial} #{lastname}"}.compact
   end
 
   def edit
     @people_collection = Person.active
```

Comment:
Given that you are re-assigning it on line 25, should we drop this line ?

Reply:
Which line are you suggesting we drop?

Reply:
All the numbers I'm using are the new line numbers. 
I suggest we get rid of line 29, as you re-assign it in line 33. The assignment at line 29 never gets used.

Reply:
It does get used, though. I'm using the assignment from line 29 and then sorting it.